### DISCUSSION TITLE
Supplying "billing_code_type_version"

### DISCUSSION THREAD

**Discussion by AJLgeHG** [CMSgov/price-transparency-guide#203] [Q&A] [answered]
The "billing_code_type_version" is a required field on the "Allowed-Amounts" machine-readable file. What are the options if your organization doesn't house this field and doesn't have it available to make available on the file? 

I've seen one suggestion to simply use the plan year, for example, "2021". Is that adequate/acceptable?

Thanks for your feedback.

**Comment by shaselton-usds** [comment, marked as answer]
@AJLgeHG Do you have the `billing_code_type` and `billing_code` when trying to build the allowed amount file?

**Reply by AJLgeHG** [reply]
Thanks. Yes, I have the billing_code_type and billing_code.

**Reply by shaselton-usds** [reply]
If there isn't a specific version associated with the `billing_code_type` like you would find with ICD, DRG, or CPT (which versions their billing codes by year as well), you can use the plan year for the version.

**Reply by AJLgeHG** [reply]
Thanks, Scott. Very much appreciated.

**Reply by angelamarie0975** [reply]
@shaselton-usds - in response to your last comment - I am just trying to make sure that I fully understand your response. In your response you stated that if we don't have versioning for codes 'like you would find with ICD, DRG, or CPT', using the plan year instead is ok, but that can be interpreted in 2 ways:

1.  If you have codes (not ICD, DRG, or CPT) and no versioning - ok to use plan year for those codes, but for ICD,DRG, or CPT codes, get the versioning from CMS/CPT
2.  If you no versioning stored for ANY billing code type, ok to use plan year instead

If we do not store the version for DRG/CPT codes in our systems, are we expected to apply the versioning for DRG/CPT codes from the files that are published publicly? If so, that means that we will need to create more logic to pull in this data to store it for MRF purposes only since we don't house that data in our system. Just want to make sure we know exactly what the expectation is.

**Reply by RLTx1391** [reply]
@shaselton-usds can we get an answer to @angelamarie0975 's question? Is number 2 what we should be going by? We did have the question specifically for using plan year for DRGs.

**Reply by shaselton-usds** [reply]
@RLTx1391 @angelamarie0975 for questions 2, you should be using the coding set's version, but if impossible to get, use the plan's year.
